test/end-to-end/core.sh

```diff
@@ -326,14 +326,22 @@ os::cmd::expect_success 'oc whoami'
 
 echo "[INFO] Running a CLI command in a container using the service account"
 os::cmd::expect_success 'oc policy add-role-to-user view -z default'
-oc run cli-with-token --attach --image="openshift/origin:${TAG}" --restart=Never -- cli status --loglevel=4 > "${LOG_DIR}/cli-with-token.log" 2>&1
```

Comment:
if `oc run --attach` doesn't work any more, we have to fix that, not work around the issue in a test